File: Rand.js
function Rand()
{
	this.c = -1.99999999999999;
	this.x = this.c;
    	this.range = this.c * -1;

	this.Get = get
}

function get()
{
	this.x = (this.x * this.x) + this.c;

	return (this.x / this.range);
}

File: Rand.cs
using System;
using System.Collections.Generic;
using System.Text;

namespace Lines2_1
{
    class Rand
    {
        private double c;
        private double x;
        private double range;

        public Rand()
        {
            c = -1.99999999999999;
            x = c;
            range = c * -1;
        }

        public double Get()
        {
            x = (x * x) + c;

            return (x / range);
        }
    }
}
